client/cypress/integration/HomePage.spec.js

```diff
@@ -0,0 +1,14 @@
+describe('My First Test', () => {
+    it('Does not do much!', () => {
```

Comment:
Better to go with proper description here, smth like:
'User should be able to sign up'

Reply:
Oh, just forgot to change the name from the cypress example)